packages/nc-gui/composables/useMultiSelect/index.ts

```diff
@@ -106,6 +115,41 @@ export function useMultiSelect(
           if (typeof textToCopy === 'object') {
```

Comment:
Above this, you can add an additional if condition to check content is a datejs instance/object and return. Just ignore my comment if it doesn't make much difference.